roles/installer/tasks/install.yml

```diff
@@ -94,48 +94,6 @@
 - name: Include resources configuration tasks
```

Comment:
```suggestion
# Includes wait for any migrations to complete
- name: Include resources configuration tasks
```

Reply:
I updated the task name to reflect its expanded role.  Thank you for the input!